build.gradle

```diff
@@ -17,6 +17,24 @@ plugins {
 
 apply from: "$rootDir/gradle/ci-support.gradle"
 
+/**
+ * Allows build cache relocatability for Delombok task
+ */
+class DelombokArgumentProvider implements CommandLineArgumentProvider {
```

Comment:
Ohhh, this is great, thank you! Is it okay to ask you to move it to https://github.com/testcontainers/testcontainers-java/tree/master/buildSrc/src/main/groovy/org/testcontainers/build ? 😊
We can do it as a follow up PR too 👍

Reply:
just done @bsideup  👍